src/tasks/tasks.js

```diff
@@ -158,14 +181,21 @@ function digStairTask(steps: number, height?: number) {
   if (steps === 0) {
     return Promise.resolve();
   } else {
-    // Dig down
-    return dig().then(() => sleep(1000))
-                .then(() => digWallTask(h))
-                // Move forward
-                .then(() => moveTo(center(bot.entity.position.offset(1,0,0))))
-                .then(() => sleep(100))
-                // Repeat
-                .then(() => digStairTask(steps-1, h));
+    var nextBlock = findSafeStandingBlockForward();
+    var nextPosition = bot.entity.position;
```

Comment:
Need... I had some problems with flow where the nextPosition/block could be null, even though i checked it.